datastream-kafka-connector/src/main/java/com/linkedin/datastream/connectors/kafka/KafkaConsumerFactoryImpl.java

```diff
@@ -17,10 +17,21 @@
  * A factory for creating Kafka {@link Consumer} instances
  */
 public class KafkaConsumerFactoryImpl implements KafkaConsumerFactory<byte[], byte[]> {
+  private static final String KEY_DESERIALIZER = ByteArrayDeserializer.class.getCanonicalName();
```

Comment:
I'd recommend prepending 'DEFAULT_' to these variable names

Reply:
@celiakung Just did 👍